api/domain/common/messages.proto

```diff
@@ -38,3 +38,8 @@ message CertificateChain {
   // Certificate
   bytes certificate = 2;
 }
+
+enum UserSource {
+  INTERNAL = 0;
+  SCIM = 1;
```

Comment:
That's only to see that this user is from an external source and which one. So that we can implement that this users are locked and can not be changed via m8 api. It is not that important where the user exactly is coming from IMHO.